src/Tizen.NET.MaterialComponents/Components/MConformant.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using ElmSharp;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MConformant : Conformant
+    {
+        Layout _layout = null;
+        Box _box = null;
+
+        public MConformant(Window parent) : base(parent)
+        {
+            _layout = new Layout(parent)
+            {
+                WeightX = 1,
+                WeightY = 1,
+                AlignmentX = -1,
+                AlignmentY = -1,
+            };
+            _layout.SetTheme("layout", Styles.MaterialApplication, "default");
+            _layout.Show();
+            base.SetContent(_layout);
+        }
+
+        public new void SetContent(EvasObject content)

```

Comment:
why don't you override `CreateHandle()` instead?

Reply:
Fixed it. Thank you.